src/NuGet.Clients/NuGetConsole.Host.PowerShell/NuGetHostUserInterface.cs

```diff
@@ -22,7 +22,7 @@ internal class NuGetHostUserInterface : PSHostUserInterface, IHostUISupportsMult
         public const ConsoleColor NoColor = (ConsoleColor)(-1);
         private const int VkCodeReturn = 13;
         private const int VkCodeBackspace = 8;
-        private static Color[] _consoleColors;
+        private static Color[] ConsoleColors;
```

Comment:
this file only has formatting improvements/etc. worth including with no real changes?